server/.golangci.yml

```diff
@@ -86,8 +86,9 @@ issues:
         channels/api4/remote_cluster.go|\
         channels/api4/role_test.go|\
         channels/api4/scheme_test.go|\
+        channels/api4/shared_channel_test.go|\
+        channels/api4/status_test.go|\
```

Comment:
That diff seems wrong

Reply:
I don't know what's causing the error. I previously solved the merged conflict too and it was all good that time. I removed these lines though.